### PR TITLE
June destination

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

Create June destination doc

### CODE REVIEW

File: docs/destinations/streaming-destinations/june/june-cloud-mode.mdx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+| `traits`/`context.traits`              | `traits`      |Object | Optional |
+| `properties`            | `properties`      | Object | Optional |
+| `context.externalId.id` (when `context.externalId.type` = `juneGroupId`/`properties.groupId`)  | `context.groupId`    | String | Optional |
+
+## Group
+
+You can use the <Link to="/event-spec/standard-events/group/">`group`</Link> call to create a new company or update an existing company's details in June.
+
+<div class="warningBlock">
+To create a company or update the company details, the <code class="inline-code">groupId</code> should be passed in the <code class="inline-code">track</code> call.
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/june/june-cloud-mode.mdx:70]: This statement is wrong. We are not creating/updating company details in track call.
We can use :- 
`If you're identifying companies, you should pass the `groupId` in track call.
`groupId`  in track call allows you to specify what company performed an event`

File: docs/destinations/streaming-destinations/june/june-cloud-mode.mdx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+RudderStack lets you send your event data to June via <Link to="/destinations/rudderstack-connection-modes/#cloud-mode">cloud mode</Link>.
+
+<div class="infoBlock">
+Find the open-source transformer code for this destination in the <a href="https://github.com/rudderlabs/rudder-transformer/tree/master/v0/destinations/june">GitHub repository</a>.
+</div>
+
+## Identify
+
+You can use the <Link to="/event-spec/standard-events/identify/">`identify`</Link> call to create new users or update existing users' details in June. RudderStack sends the `identify` call to June as an `identify` event.
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/june/june-cloud-mode.mdx:14]: Can attach `JUNE` Identify API link here.
https://www.june.so/docs/api#:~:text=Copy-,Identifying%20users,-You%20can%20use

File: docs/destinations/streaming-destinations/june/june-cloud-mode.mdx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+| RudderStack property   | June property  | Data type | Presence |
+| :-------------| :-----------------|:-------------| :-----------------|
+| `userId`/`traits.userId`/`traits.id`/`context.traits.userId`/`context.traits.id`       | `userId`        | String | Required |
+| `originalTimestamp`/`timestamp`  | `timestamp`      |String *(ISO 8601 date format yyyy-MM-ddTHH:mm:ss.SSSZ)* | Required |
+| `traits`/`context.traits`              | `traits`      |Object | Optional |
+
+
+## Track
+
+The <Link to="/event-spec/standard-events/track">`track`</Link> call lets you capture user events along with the properties associated with them. RudderStack sends the `track` call to June as a `track` event.
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/june/june-cloud-mode.mdx:38]: Can attach JUNE Track API link here.
https://www.june.so/docs/api#:~:text=Copy-,Send%20track%20events,-In%20order%20to

File: docs/destinations/streaming-destinations/june/june-cloud-mode.mdx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+| `userId`/`traits.userId`/`traits.id`/`context.traits.userId`/`context.traits.id`       | `userId`        | String | Required |
+| `event`            | `event`      | String | Required |
+| `originalTimestamp`/`timestamp`  | `timestamp`      |String *(ISO 8601 date format yyyy-MM-ddTHH:mm:ss.SSSZ)* | Required |
+| `traits`/`context.traits`              | `traits`      |Object | Optional |
+| `properties`            | `properties`      | Object | Optional |
+| `context.externalId.id` (when `context.externalId.type` = `juneGroupId`/`properties.groupId`)  | `context.groupId`    | String | Optional |
+
+## Group
+
+You can use the <Link to="/event-spec/standard-events/group/">`group`</Link> call to create a new company or update an existing company's details in June.
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/june/june-cloud-mode.mdx:67]: Can attach `JUNE` group API link here
https://www.june.so/docs/api#:~:text=Identifying%20companies%20(optional)

**Review by Gauravudia — CHANGES_REQUESTED** [review]

File: docs/destinations/streaming-destinations/june/june-cloud-mode.mdx [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+
+## Track
+
+The <Link to="/event-spec/standard-events/track">`track`</Link> call lets you capture user events along with the properties associated with them. RudderStack sends the `track` call to June as a `track` event.
+
+A sample `track` call is shown below:
+
+```javascript
+rudderanalytics.track("Product Reviewed", {
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/june/june-cloud-mode.mdx:43]: Please use the update snippet from research spec.

File: docs/destinations/streaming-destinations/june/june-device-mode.mdx [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+The <Link to="/event-spec/standard-events/track">`track`</Link> call lets you capture user events along with the properties associated with them. RudderStack sends the `track` call to June as a `track` event.
+
+<div class="warningBlock">
+Make sure that you make an <code class="inline-code">identify</code> call before a <code class="inline-code">track</code> call.
+</div>
+
+A sample `track` call is shown below:
+
+```javascript
+rudderanalytics.track("Product Reviewed", {
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/june/june-device-mode.mdx:46]: Please use the updated snippet from research spec.

File: docs/destinations/streaming-destinations/june/june-device-mode.mdx [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+
+### Property mappings
+
+The following table lists the property mappings between RudderStack and June for the `track` call:
+
+| RudderStack property   | June property  | Data type | Presence |
+| :-------------| :-----------------|:-------------| :-----------------|
+| `event`            | `event`      | String | Required |
+| `properties`            | `properties`      | Object | Optional |
+| `context.externalId.id` (when `context.externalId.type` = `juneGroupId`/`properties.groupId`)   | `context.groupId`    | String | Optional |
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/june/june-device-mode.mdx:63]: same as [here](https://github.com/rudderlabs/rudderstack-docs/pull/919/files#r980965089)

File: docs/destinations/streaming-destinations/june/june-device-mode.mdx [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+| :-------------| :-----------------|:-------------| :-----------------|
+| `name`            | `name`      | String | Optional |
+| `properties`            | `properties`      | Object | Optional |
+
+## Group
+
+You can use the <Link to="/event-spec/standard-events/group/">`group`</Link> call to create a new company or update an existing company's details in June.
+
+<div class="warningBlock">
+To create a company or update the company details, the <code class="inline-code">groupId</code> should be passed in the <code class="inline-code">track</code> call.
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/june/june-device-mode.mdx:90]: same as [here](https://github.com/rudderlabs/rudderstack-docs/pull/919/files#r980982234)

**Review by Gauravudia — CHANGES_REQUESTED** [review]

File: docs/destinations/streaming-destinations/june/june-cloud-mode.mdx [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+| `originalTimestamp`/`timestamp`  | `timestamp`      |String *(ISO 8601 date format yyyy-MM-ddTHH:mm:ss.SSSZ)* | Required |
+| `properties`            | `properties`      | Object | Optional |
+| <ul><li>If `context.externalId.id` (when `context.externalId.type` = `juneGroupId`) is present.</li><li>Otherwise, `properties.groupId`.</li></ul>   | `context.groupId`    | String | Optional |
+
+## Group
+
+You can use the <Link to="/event-spec/standard-events/group/">`group`</Link> call to create a new company or update an existing company's details in June. Refer to the [June documentation](https://www.june.so/docs/api#:~:text=Identifying%20companies%20(optional)) for more information.
+
+<div class="warningBlock">
+To identify a company, pass the <code class="inline-code">groupId</code> in the <code class="inline-code">track</code> call. It allows you to specify which company performed that specific event.
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/june/june-cloud-mode.mdx:85]: - Statement seems incorrect. We are not `identifying companies `in track call.
`Group` call is used to identify company. It is recommended to pass `groupId` in `track` call If user is using the group call (Identify company)
- Also, please move this `note` to track call.

File: docs/destinations/streaming-destinations/june/june-device-mode.mdx [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+| :-------------| :-----------------|:-------------| :-----------------|
+| `name`            | `name`      | String | Optional |
+| `properties`            | `properties`      | Object | Optional |
+
+## Group
+
+You can use the <Link to="/event-spec/standard-events/group/">`group`</Link> call to create a new company or update an existing company's details in June. Refer to the [June documentation](https://www.june.so/docs/api#:~:text=Identifying%20companies%20(optional)) for more information.
+
+<div class="warningBlock">
+To identify a company, pass the <code class="inline-code">groupId</code> in the <code class="inline-code">track</code> call. It allows you to specify which company performed that specific event.
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/june/june-device-mode.mdx:106]: - Statement seems incorrect. We are not `identifying companies` in track call.
`Group` call is used to identify company. It is recommended to pass `groupId` in track call If user is using the group call (Identify company)
- Also, please move this `note` to track call.

**Review by Gauravudia — CHANGES_REQUESTED** [review]

**Review by Gauravudia — APPROVED** [review]

File: docs/destinations/streaming-destinations/june/index.mdx [hunk 11]
```diff
@@ -0,0 +1,14 @@
+---
+title: "June"
+description: Step-by-step guide on sending your event data from RudderStack to June.
+---
+
+[June](https://www.june.so/)is a lightweight product analytics tool that helps you define and measure product usage intuitively by tracking user retention, active users, acquisition funnel, engagement, feature usage, etc.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/june/index.mdx:6]:
```suggestion
[June](https://www.june.so/) is a lightweight product analytics tool that helps you define and measure product usage intuitively by tracking user retention, active users, acquisition funnel, engagement, feature usage, etc.
```

File: docs/destinations/streaming-destinations/june/setting-up-june-in-rudderstack.mdx [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+| Cloud Mode    | Supported | Supported  | Supported  |
+
+Once you have confirmed that the source platform supports sending events to June, follow these steps:
+
+1. From your [RudderStack dashboard](https://app.rudderstack.com/), add the source. Then, from the list of destinations, select **June**.
+2. Assign a name to your destination and click **Continue**.
+
+## Connection settings
+
+To successfully configure June as a destination, configure the following settings:
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/june/setting-up-june-in-rudderstack.mdx:20]:
```suggestion
To successfully set up June as a destination, configure the following settings:
```

Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/june/setting-up-june-in-rudderstack.mdx:20]: Made a similar change in our destination template, as "configure" is repeated here.

File: docs/destinations/streaming-destinations/june/setting-up-june-in-rudderstack.mdx [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+- **API Key**: Enter the API key from the June dashboard. Refer to the <Link to="#faq">FAQ</Link> section for more information on obtaining the API key. 
+
+- **Client-side Events Filtering**: This setting lets you specify which events should be blocked or allowed to flow through to June. For more information on this setting, refer to the <Link to="/sources/sdks/event-filtering/">Client-side Events Filtering</Link> guide.
+    
+
+## FAQ
+
+### How do I obtain the June API key?
+
+1. Login to the [June dashboard](https://www.june.so/).
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/june/setting-up-june-in-rudderstack.mdx:31]:
```suggestion
1. Login to the [June dashboard](https://analytics.june.so/login).
```

File: docs/destinations/streaming-destinations/june/setting-up-june-in-rudderstack.mdx [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+2. Click on your workspace and select **Settings & Integrations**.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/june/setting-up-june-in-rudderstack.mdx:32]:
```suggestion
2. Go to your workspace and select **Settings & Integrations**.
```

Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/june/setting-up-june-in-rudderstack.mdx:32]: Would this be accurate?

File: docs/destinations/streaming-destinations/june/june-cloud-mode.mdx [hunk 15]
```diff
@@ -0,0 +1,118 @@
+---
+title: "June Cloud Mode"
+description:  Detailed technical documentation on sending events to June using the RudderStack cloud mode.
+---
+
+RudderStack lets you send your event data to June via <Link to="/destinations/rudderstack-connection-modes/#cloud-mode">cloud mode</Link>.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/june/june-cloud-mode.mdx:6]:
```suggestion
RudderStack lets you send your event data to June via the <Link to="/destinations/rudderstack-connection-modes/#cloud-mode">cloud mode</Link>.
```

File: docs/destinations/streaming-destinations/june/june-cloud-mode.mdx [hunk 16]
```diff
@@ -0,0 +1,118 @@
...
+
+<div class="infoBlock">
+Find the open-source transformer code for this destination in the <a href="https://github.com/rudderlabs/rudder-transformer/tree/master/v0/destinations/june">GitHub repository</a>.
+</div>
+
+## Identify
+
+You can use the <Link to="/event-spec/standard-events/identify/">`identify`</Link> call to create new users or update existing users' details in June. RudderStack sends the `identify` call to June as an [`identify`](https://www.june.so/docs/api#:~:text=Copy-,Identifying%20users,-You%20can%20use) event.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/june/june-cloud-mode.mdx:14]:
```suggestion
You can use the <Link to="/event-spec/standard-events/identify/">`identify`</Link> call to create new users or update an existing user's details in June. RudderStack sends the `identify` call to June as an [`identify`](https://www.june.so/docs/api#:~:text=Copy-,Identifying%20users,-You%20can%20use) event. 
```

File: docs/destinations/streaming-destinations/june/june-cloud-mode.mdx [hunk 17]
```diff
@@ -0,0 +1,118 @@
...
+```
+
+### Property mappings
+
+The following table lists the property mappings between RudderStack and June for the `identify` call:
+
+| RudderStack property   | June property  | Data type | Presence |
+| :-------------| :-----------------|:-------------| :-----------------|
+| `userId`/`traits.userId`/`traits.id`/`context.traits.userId`/`context.traits.id`       | `userId`        | String | Required |
+| `originalTimestamp`/`timestamp`  | `timestamp`      |String *(ISO 8601 date format yyyy-MM-ddTHH:mm:ss.SSSZ)* | Required |
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/june/june-cloud-mode.mdx:32]:
```suggestion
| `originalTimestamp`/`timestamp`  | `timestamp`      |String (ISO 8601 date format `yyyy-MM-ddTHH:mm:ss.SSSZ`) | Required |
```

File: docs/destinations/streaming-destinations/june/june-cloud-mode.mdx [hunk 18]
```diff
@@ -0,0 +1,118 @@
...
+| `traits`/`context.traits`              | `traits`      |Object | Optional |
+
+
+## Track
+
+The <Link to="/event-spec/standard-events/track">`track`</Link> call lets you capture user events along with the properties associated with them. RudderStack sends the `track` call to June as a [`track`](https://www.june.so/docs/api#:~:text=Copy-,Send%20track%20events,-In%20order%20to) event.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/june/june-cloud-mode.mdx:38]:
```suggestion
The <Link to="/event-spec/standard-events/track">`track`</Link> call lets you capture user events along with the properties associated with them. RudderStack sends the `track` call to June using their [`track`](https://www.june.so/docs/api#:~:text=Copy-,Send%20track%20events,-In%20order%20to) API.
```

File: docs/destinations/streaming-destinations/june/setting-up-june-in-rudderstack.mdx (hunk 14 above)
Comment (ishachopra0401) [reply docs/destinations/streaming-destinations/june/setting-up-june-in-rudderstack.mdx:32]: Actually you dont need to click on the workspace, it is there by default

File: docs/destinations/streaming-destinations/june/june-cloud-mode.mdx [hunk 19]
```diff
@@ -0,0 +1,118 @@
...
+
+## FAQ
+
+### Can I make an `identify` or `track` call using `anonymousId`?
+
+No, you can't. June does not stitches identities together or offers anonymous user merging. It is rather used for logged-in experiences where you can measure user activation and retention.
+
+### My user is created as `Unknown` in June dashboard. What might be the reason?
+
+June categorizes a user as `Unknown` if the `firsName`, `lastName`, `name`, `username`, or `email` is not present in the `identify` call's `traits`.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/june/june-cloud-mode.mdx:118]:
```suggestion
June categorizes a user as `Unknown` if the `firstName`, `lastName`, `name`, `username`, or `email` is not present in the `traits` object of the `identify` call.
```

File: docs/destinations/streaming-destinations/june/june-cloud-mode.mdx [hunk 20]
```diff
@@ -0,0 +1,118 @@
...
+| `userId`/`traits.userId`/`traits.id`/`context.traits.userId`/`context.traits.id`       | `userId`        | String | <ul><li>Required while creating a company.</li><li>Optional while updating the company details.</li></ul>  |
+| `originalTimestamp`/`timestamp`  | `timestamp`      |DateTime *(ISO 8601 date format yyyy-MM-ddTHH:mm:ss.SSSZ)* | Required |
+| `groupId`/`traits.groupId`              | `groupId`      |String | Required |
+| `traits`/`context.traits`              | `traits`      |Object | Optional |
+
+## FAQ
+
+### Can I make an `identify` or `track` call using `anonymousId`?
+
+No, you can't. June does not stitches identities together or offers anonymous user merging. It is rather used for logged-in experiences where you can measure user activation and retention.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/june/june-cloud-mode.mdx:114]:
```suggestion
No, you can't. June does not stitch identities together or support merging anonymous users. Rather, it is best used to measure user activation and retention when the user is logged in.
```

Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/june/june-cloud-mode.mdx:114]: LMK if this change makes sense.

File: docs/destinations/streaming-destinations/june/june-cloud-mode.mdx [hunk 21]
```diff
@@ -0,0 +1,118 @@
...
+});
+```
+
+### Property mappings
+
+The following table lists the property mappings between RudderStack and June for the `group` call:
+
+| RudderStack property   | June property  | Data type | Presence |
+| :-------------| :-----------------|:-------------| :-----------------|
+| `userId`/`traits.userId`/`traits.id`/`context.traits.userId`/`context.traits.id`       | `userId`        | String | <ul><li>Required while creating a company.</li><li>Optional while updating the company details.</li></ul>  |
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/june/june-cloud-mode.mdx:105]:
```suggestion
| `userId`/`traits.userId`/`traits.id`/`context.traits.userId`/`context.traits.id`       | `userId`        | String | <ul><li>Required for creating a company.</li><li>Optional for updating the company details.</li></ul>  |
```

File: docs/destinations/streaming-destinations/june/june-cloud-mode.mdx [hunk 22]
```diff
@@ -0,0 +1,118 @@
...
+| `originalTimestamp`/`timestamp`  | `timestamp`      |DateTime *(ISO 8601 date format yyyy-MM-ddTHH:mm:ss.SSSZ)* | Required |
+| `properties`            | `properties`      | Object | Optional |
+| <ul><li>If `context.externalId.id` (when `context.externalId.type` = `juneGroupId`) is present.</li><li>Otherwise, `properties.groupId`.</li></ul>   | `context.groupId`    | String | Optional |
+
+## Group
+
+You can use the <Link to="/event-spec/standard-events/group/">`group`</Link> call to create a new company or update an existing company's details in June. Refer to the [June documentation](https://www.june.so/docs/api#:~:text=Identifying%20companies%20(optional)) for more information.
+
+<div class="warningBlock">
+If you are using a <code class="inline-code">group</code> call to identify a company, it is recommended to pass the <code class="inline-code">groupId</code> in the <code class="inline-code">track</code> call. It allows you to specify which company performed that specific event.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/june/june-cloud-mode.mdx:85]:
```suggestion
If you are using a <code class="inline-code">group</code> call to identify a company, it is recommended to pass the <code class="inline-code">groupId</code> in the <code class="inline-code">track</code> call. It allows you to specify which company performed that specific event. For more information, refer to the <a href="https://www.june.so/docs/api#:~:text=Copy-,Send%20track%20events,-In%20order%20to">June documentation</a>.
```

File: docs/destinations/streaming-destinations/june/june-device-mode.mdx [hunk 23]
```diff
@@ -0,0 +1,143 @@
...
+| `userId`/`context.traits.userId`/`context.traits.id`       | `userId`        | String | Required |
+| `context.traits`              | `traits`      |Object | Optional |
+
+
+## Track
+
+The <Link to="/event-spec/standard-events/track">`track`</Link> call lets you capture user events along with the properties associated with them. RudderStack sends the `track` call to June as a [`track`](https://www.june.so/docs/api#:~:text=Copy-,Send%20track%20events,-In%20order%20to) event.
+
+<div class="warningBlock">
+Make sure that you make an <code class="inline-code">identify</code> call before a <code class="inline-code">track</code> call.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/june/june-device-mode.mdx:40]: Should we include this information in the cloud mode doc too? 
From the FAQ, I understood this destination works best with identified users, so this is applicable for both modes.

File: docs/destinations/streaming-destinations/june/june-device-mode.mdx [hunk 24]
```diff
@@ -0,0 +1,143 @@
...
+| :-------------| :-----------------|:-------------| :-----------------|
+| `name`            | `name`      | String | Optional |
+| `properties`            | `properties`      | Object | Optional |
+
+## Group
+
+You can use the <Link to="/event-spec/standard-events/group/">`group`</Link> call to create a new company or update an existing company's details in June. Refer to the [June documentation](https://www.june.so/docs/api#:~:text=Identifying%20companies%20(optional)) for more information.
+
+<div class="warningBlock">
+If you are using a <code class="inline-code">group</code> call to identify a company, it is recommended to pass the <code class="inline-code">groupId</code> in the <code class="inline-code">track</code> call. It allows you to specify which company performed that specific event.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/june/june-device-mode.mdx:106]:
```suggestion
If you are using a <code class="inline-code">group</code> call to identify a company, it is recommended to pass the <code class="inline-code">groupId</code> in the <code class="inline-code">track</code> call. It allows you to specify which company performed that specific event. For more information, refer to the <a href="https://www.june.so/docs/api#:~:text=Copy-,Send%20track%20events,-In%20order%20to">June documentation</a>.
```

File: docs/destinations/streaming-destinations/june/june-device-mode.mdx [hunk 25]
```diff
@@ -0,0 +1,143 @@
...
+</div>
+
+<div class="warningBlock">
+Make sure that you make an <code class="inline-code">identify</code> call before a <code class="inline-code">group</code> call to create a company successfully.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/june/june-device-mode.mdx:110]: Applicable for the cloud mode doc too. An alternative would be to just specify in the `identify` section that the user needs to make an `identify` call before making any `track`, `group`, or `page` calls.

File: docs/destinations/streaming-destinations/june/june-device-mode.mdx [hunk 26]
```diff
@@ -0,0 +1,143 @@
...
+| RudderStack property   | June property  | Data type | Presence |
+| :-------------| :-----------------|:-------------| :-----------------|
+| `groupId`/`traits.groupId`            | `groupId`      | String | Required |
+| `traits`            | `traits`      | Object | Optional |
+
+## FAQ
+
+### Can I make an `identify` or `track` call using `anonymousId`?
+
+No, you can't. June does not stitches identities together or offers anonymous user merging. It is rather used for logged-in experiences where you can measure user activation and retention.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/june/june-device-mode.mdx:136]:
```suggestion
No, you can't. June does not stitch identities together or support merging anonymous users. Rather, it is best used to measure user activation and retention when the user is logged in.
```

File: docs/destinations/streaming-destinations/june/june-device-mode.mdx [hunk 27]
```diff
@@ -0,0 +1,143 @@
...
+
+### My user is created as `Unknown` in June dashboard. What might be the reason?
+
+June categorizes a user as `Unknown` if the `firsName`, `lastName`, `name`, `username`, or `email` is not present in the `identify` call's `traits`.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/june/june-device-mode.mdx:140]:
```suggestion
June categorizes a user as `Unknown` if the `firstName`, `lastName`, `name`, `username`, or `email` is not present in the `traits` object of the `identify` call.
```

File: docs/destinations/streaming-destinations/june/june-device-mode.mdx (hunk 25 above)
Comment (Gauravudia) [reply docs/destinations/streaming-destinations/june/june-device-mode.mdx:110]: Yes, We can add a note in `Identify` call.

Comment (ishachopra0401) [reply docs/destinations/streaming-destinations/june/june-device-mode.mdx:110]: @Gauravudia I remember we had a discussion over this. [Ref](https://github.com/rudderlabs/rudderstack-docs/pull/919/commits/df0d615d711cb421d24882b460746e5038588a8b#r980968061)

Also, another one for track call in device mode. [Ref](https://www.notion.so/June-Cloud-and-Device-mode-Destination-1e02191a9d0e4f579c5ec2cd2609cb27?d=a514b1c39eaa4f5b8950ae9969229320#8dccdfd4b61f476c945ca5acb48f2503)

Can you please confirm again if the latest note is valid for both the modes?

Comment (Gauravudia) [reply docs/destinations/streaming-destinations/june/june-device-mode.mdx:110]: @ishachopra0401 Above `note` is valid for both modes. We can add a `note` in Identify call as suggested by Amey.

**Review by ameypv-rudder — APPROVED** [review]

File: docs/destinations/streaming-destinations/june/june-device-mode.mdx [hunk 28]
```diff
@@ -0,0 +1,136 @@
...
+
+RudderStack lets you send your event data to June destination via the <Link to="/destinations/rudderstack-connection-modes/#device-mode">device mode</Link> using the native web SDK.
+
+<div class="infoBlock">
+Find the open-source code for this destination in the <a href="https://github.com/rudderlabs/rudder-sdk-js/tree/production/integrations/june">GitHub repository</a>.
+</div>
+
+## Identify
+
+You can use the <Link to="/event-spec/standard-events/identify/">`identify`</Link> call to create new users or update existing users' details in June. RudderStack sends the `identify` call to June as an [`identify`](https://www.june.so/docs/api#:~:text=Copy-,Identifying%20users,-You%20can%20use) event.
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/june/june-device-mode.mdx:14]: Link pointing to Rest API endpoint.
sdk identify call doc link :- https://www.june.so/docs/identifying-users

File: docs/destinations/streaming-destinations/june/june-device-mode.mdx [hunk 29]
```diff
@@ -0,0 +1,136 @@
...
+
+| RudderStack property   | June property  | Data type | Presence |
+| :-------------| :-----------------|:-------------| :-----------------|
+| `userId`/`context.traits.userId`/`context.traits.id`       | `userId`        | String | Required |
+| `context.traits`              | `traits`      |Object | Optional |
+
+
+## Track
+
+The <Link to="/event-spec/standard-events/track">`track`</Link> call lets you capture user events along with the properties associated with them. RudderStack sends the `track` call to June as a [`track`](https://www.june.so/docs/api#:~:text=Copy-,Send%20track%20events,-In%20order%20to) event.
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/june/june-device-mode.mdx:41]: Link pointing to Rest API endpoint. 
sdk track event doc link :- https://www.june.so/docs/tracking-events

File: docs/destinations/streaming-destinations/june/june-device-mode.mdx [hunk 30]
```diff
@@ -0,0 +1,136 @@
...
+The following table lists the property mappings between RudderStack and June for the `page` call:
+
+| RudderStack property   | June property  | Data type | Presence |
+| :-------------| :-----------------|:-------------| :-----------------|
+| `name`            | `name`      | String | Optional |
+| `properties`            | `properties`      | Object | Optional |
+
+## Group
+
+You can use the <Link to="/event-spec/standard-events/group/">`group`</Link> call to create a new company or update an existing company's details in June. Refer to the [June documentation](https://www.june.so/docs/api#:~:text=Identifying%20companies%20(optional)) for more information.
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/june/june-device-mode.mdx:103]: Link pointing to Rest endpoint doc.
sdk group call doc link :- https://www.june.so/docs/tracking-groups

File: docs/destinations/streaming-destinations/june/june-device-mode.mdx [hunk 31]
```diff
@@ -0,0 +1,136 @@
...
+
+<div class="warningBlock">
+If you are using a <code class="inline-code">group</code> call to identify a company, it is recommended to pass the <code class="inline-code">groupId</code> in the <code class="inline-code">track</code> call. It allows you to specify which company performed that specific event. For more information, refer to the <a href="https://www.june.so/docs/api#:~:text=Copy-,Send%20track%20events,-In%20order%20to">June documentation</a>.
```
Comment (Gauravudia) [inline docs/destinations/streaming-destinations/june/june-device-mode.mdx:106]: Link pointing to Rest endpoint doc.
sdk group call doc link :- https://www.june.so/docs/tracking-groups

**Pull request merged**